### PR TITLE
vp-2443 cart api integration tests

### PR DESCRIPTION
**Pull request by evanbtlr**

### CODE REVIEW

**Comment by lnetrebskii** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-3378 has been created

File: VirtoCommerce.Storefront/Properties/serviceDependencies.local.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (YanBrovkin) [inline VirtoCommerce.Storefront/Properties/serviceDependencies.local.json:1]: Please remove - it's for local use

File: VirtoCommerce.Storefront.IntegrationTests/Cart/ApiCartControllerTests.cs [hunk 2]
```diff
@@ -74,22 +78,61 @@ public async Task GetCart_IfCartExistForUser_ShouldReturnExistedCart()
...
-                .InsertCartItem(new AddCartItem { Id = "9cbd8f316e254a679ba34a900fccb076", Quantity = 1 });
+                .ClearCart()
+                .InsertCartItem(new AddCartItem { Id = Product.Quadcopter, Quantity = 1 });
 
             //act
-            var result = await GetCart(_client);
+            var result = await _client.GetCart();
 
             //assert
             GetCartComparationResult(
```
Comment (YanBrovkin) [inline VirtoCommerce.Storefront.IntegrationTests/Cart/ApiCartControllerTests.cs:88]: Expected string to be <null>, but found "{
      "isAnonymous": [
        true,
        false
      ],
      "customerName": [
        "Anonymous",
        "admin"
      ],
      "type": [
        "cart",
        0,
        0
      ]
    }"

File: VirtoCommerce.Storefront.IntegrationTests/Cart/ApiCartControllerTests.cs [hunk 3]
```diff
@@ -112,50 +155,55 @@ public async Task GetCartItemsCount_IfCartHasNoItems_ShouldReturnZero()
...
+            _client
+                .ClearCart();
 
             //act
-            var result = await GetCart(_client);
+            var result = await _client.GetCart();
 
             //assert
-            GetCartComparationResult(result, "GetEmptyCartForAdmin")
+            GetCartComparationResult(result, "GetEmptyCartForAdmin", new[] { "$" }, new[] { "id" })
```
Comment (YanBrovkin) [inline VirtoCommerce.Storefront.IntegrationTests/Cart/ApiCartControllerTests.cs:190]: Expected string to be <null>, but found "{
      "isAnonymous": [
        true,
        false
      ],
      "customerName": [
        "Anonymous",
        "admin"
      ],
      "type": [
        "cart",
        0,
        0
      ]
    }

**Pull request merged**